jms/src/main/scala/akka/stream/alpakka/jms/JmsMessages.scala

```diff
@@ -41,6 +65,30 @@ object JmsType {
   def create(jmsType: String) = JmsType(jmsType)
 }
 
+object JmsTimeToLive {
+
+  /**
+   * Java API: create  [[JmsTimeToLive]]
+   */
+  def create(timeInMillis: Long) = JmsTimeToLive(timeInMillis)
```

Comment:
you may add an `apply` method taking `scala.concurrent.duration.Duration` and possibly a Java version with `TimeUnit`

Reply:
Fixed